bazel/external/requirements.in

```diff
@@ -25,7 +25,7 @@ bravado_core
 jsonschema==3.2.0
 psutil
 systemd-python
-cryptography
```

Comment:
nitpick: I think it would be nice to add a short comment with the reason why we need this restriction. Also, some more context in the PR description would be good.

Reply:
context in PR was in the writing during review :) comment in requirements.in, yes, good idea

Reply:
done